docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -0,0 +1,58 @@
+# Creating a new network from the Cleura Cloud Control Panel
+
+Before creating a server in {{extra.brand}} Cloud, you need to have a 
+network to make the new server a member of. You may have more than one 
+network per region, so let us now walk through creating a new network 
+using the {{extra.brand}} Cloud Control Panel. Fire up your favorite web 
+browser, navigate to the [Cleura Cloud](https://cleura.cloud) page, and 
+login into your {{extra.brand}} account.
+
+On the top right-hand side of the {{extra.brand}} Cloud Control Panel, 
+click the _Create_ button. A new pane will slide into from the 
+right-hand side of the browser window, titled ---unsurprisingly 
+enough--- _Create_.
+
+![Create a new object](assets/new-net-panel/shot-01.png)
```

Comment:
In the screenshots, you will want to blur the project name (in the GUI element labeled _Project,_ at the top of the screen).